lib/puppet/provider/rabbitmq_binding/rabbitmqadmin.rb

```diff
@@ -10,7 +10,7 @@
     environment HOME: '/tmp'
   end
 
-  defaultfor feature: :posix
+  confine :feature => :posix
```

Comment:
the linter isn't happy:
https://travis-ci.org/voxpupuli/puppet-rabbitmq/jobs/312947453

Maybe we should do:
`confine feature: :posix`